ajax/libs/Captionator/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Captionator",
+  "filename": "js/captionator-min.js",
+  "description": "Simple closed-captioning polyfill for HTML5. Just 8KB when gzipped!",
+  "version": "0.5",
+  "homepage": "http://captionatorjs.com/",
+  "author": {
+    "name": "Christopher Giffard",
+    "url": "https://github.com/cgiffard"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cgiffard/Captionator.git"
+  },
+  "keywords": [
+    "Captionator",
+    "captionator.js",
+    "js"
+  ],
+  "license": "BSD-2-Clause",
```

Comment:
Okay, next time you should add this info in the first commit, so that the review can be faster and efficient.